guides/doc-Managing_Hosts/topics/creating-an-insights-plan.adoc

```diff
@@ -7,15 +7,19 @@ With {ProjectX}, you can create a Red{nbsp}Hat Insights remediation plan and run
 
 To create the plan, complete the following steps:
 
-. In the {ProjectWebUI}, navigate to *Insights* > *Inventory*, and select the hosts that you want to include in an Insights plan.
-. From the *Actions* list, select *Create a new Plan/Playbook*.
-. From the Plan/Playbook Builder window, select *Create new plan* and enter a name for the plan.
-. Select whether you want the rule to apply to a specific system, group, or all systems.
-. Select one or more rules that you want to add to the plan.
+In the {ProjectWebUI}, navigate to *Configure* *>* *Insights*.
+. On the *Red Hat Insights* page, select the number of recommendations that you want to include in an Insights plan.
++
+You can only select the recommendations that have an associated playbook.
++
+. Click *Remediate*.
+. In the *Remediation Summary* window, you can select the *Resolutions* to apply.
 Use the *Filter* field to search for specific keywords.
-. Click *Save*.
-. In the Planner window, select *Run Playbook*.
+. Click *Remediate*.
+. In the *Job Invocation* page, do not change the contents of precompleted fields.
+. Optionally, for more advanced configuration of the Remote Execution Job you can click *Show Advanced Fields*.
+. Select the *Type of query* you require.
+. Select the *Schedule* you require.
+. Click *Submit*.
 
-In the Jobs window, you can view the progress of your plan as the playbook runs.
-
-You can view the Insights plan by navigating to *Insights* > *Planner*.
+In the Jobs window, you can view the progress of your plan.
```

Comment:
Can someone confirm this @ares ?

Reply:
It was originally confirmed by Ondrej Prazak